packages/utils/src/get-awe-list.ts

```diff
@@ -0,0 +1,159 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { MetriportMedicalApi, PatientDTO } from "@metriport/api-sdk";
+import { executeAsynchronously } from "@metriport/core/util/concurrency";
+import { getEnvVar, getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { out } from "@metriport/core/util/log";
+import { sleep } from "@metriport/core/util/sleep";
+import fs from "fs";
+import dayjs from "dayjs";
+import { Condition } from "@medplum/fhirtypes";
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const apiUrl = getEnvVarOrFail("API_URL");
+const delayTime = parseInt(getEnvVar("BULK_INSERT_DELAY_TIME") ?? "200");
+const numberOfParallelExecutions = parseInt(getEnvVar("PARALLEL_QUERIES") ?? "10");
+const csvHeader = "id,firstName,lastName,dob,genderAtBirth,latestAWEDate,aweInPastYear\n";
+const curDateTime = new Date();
+const runName = `Z00s-${curDateTime.toISOString()}`;
+const baseDir = `./${runName}`;
+const patientWhitelist: string[] = [];
+
+function buildCSVFileName(): string {
+  return `${baseDir}/_Z00s.csv`;
+}
+
+const metriportAPI = new MetriportMedicalApi(apiKey, {
+  baseAddress: apiUrl,
+});
+
+// find latest Z00 codes per patient
+async function getAllAWEs() {
+  fs.mkdirSync(baseDir);
+  const csvFileName = buildCSVFileName();
+  fs.writeFileSync(csvFileName, csvHeader);
+  const errors: { patientId: string; error: Error }[] = [];
+
+  const facilities = await metriportAPI.listFacilities();
+  for (const facility of facilities) {
+    let patients = await metriportAPI.listPatients(facility.id);
+    patients = patients.filter(p => patientWhitelist.includes(p.id));
```

Comment:
It'd be good to use an approach like this one, if this can be based off that branch an wait for it to be merged: https://github.com/metriport/metriport/pull/1019/files#diff-a7c4b68f53629031d1e1a1735020c853a99c9602b234c75b7559db1a1b9abdcfR140

So we have a list of patientIds in case we want to limit the search, and not worrying about facility.